### PR TITLE
Save Events to the Calendar

### PR DESCRIPTION
**Pull request by pandilladivya**
@sandarumk @HimanshiJain 
Kindly review!
image

### CODE REVIEW

**Comment by pandilladivya** [comment]
@sandarumk @HimanshiJain 
Also added Navigate to the event commit to this PR.
Do you want me to separate it?

**Comment by HimanshiJain** [comment]
I think it would be better if we separate it.

**Comment by pandilladivya** [comment]
@HimanshiJain Separated navigation from this PR!

File: app/src/main/res/layout/activity_sign_up.xml [hunk 1]
```diff
@@ -97,6 +97,7 @@
 
         </LinearLayout>
 
+
```
Comment (sandarumk) [inline app/src/main/res/layout/activity_sign_up.xml]: Any reason for the extra lines?

File: app/src/main/res/layout/content_event_detail_view.xml [hunk 2]
```diff
@@ -13,6 +13,11 @@
         android:layout_width="fill_parent"
         android:textAlignment="center"
         android:id="@+id/coordinator_layout">
+    <ScrollView
+        android:layout_width="fill_parent"
+        android:layout_height="fill_parent">
+
```
Comment (sandarumk) [inline app/src/main/res/layout/content_event_detail_view.xml]: here as well?

**Comment by sandarumk** [comment]
As a general practice please add a comment on each PR whether that PR is independent or dependent on another open PR.

File: app/src/main/res/layout/content_event_detail_view.xml (hunk 2 above)
Comment (HimanshiJain) [reply app/src/main/res/layout/content_event_detail_view.xml]: And do the indentation right for the layout files.

File: app/src/main/java/vola/systers/com/android/adapter/ScheduleEventsListAdapter.java [hunk 3]
```diff
@@ -32,6 +40,49 @@ public ScheduleEventsListAdapter(ArrayList<Event> data, Context context) {
...
+        switch (v.getId())
+        {
+            case R.id.calendar:
+                Calendar cal = Calendar.getInstance();
+                Intent intent = new Intent(Intent.ACTION_EDIT);
+                intent.setType("vnd.android.cursor.item/event");
+                String [] start_date = event.getStartDate().split("-");
+                String [] start_time = event.getStartTime().split(":");
+                cal.set(
+                        Integer.parseInt(start_date[0]),    // Year
```
Comment (HimanshiJain) [inline app/src/main/java/vola/systers/com/android/adapter/ScheduleEventsListAdapter.java]: If we are using the date and time often as a combination of  three parts each, do you think making classes -  Date and Time will make the code look neater and easier to understand? Like we did for Events?

File: app/src/main/res/layout/activity_sign_up.xml (hunk 1 above)
Comment (pandilladivya) [reply app/src/main/res/layout/activity_sign_up.xml]: No. Will remove

File: app/src/main/res/layout/content_event_detail_view.xml (hunk 2 above)
Comment (pandilladivya) [reply app/src/main/res/layout/content_event_detail_view.xml]: okay

File: app/src/main/java/vola/systers/com/android/adapter/ScheduleEventsListAdapter.java (hunk 3 above)
Comment (pandilladivya) [reply app/src/main/java/vola/systers/com/android/adapter/ScheduleEventsListAdapter.java]: Agreed. Will add a new class 😊

**Comment by pandilladivya** [comment]
@sandarumk @HimanshiJain 
Done with all the changes.
Please review and let me know if there are any more changes to be made

**Comment by sandarumk** [comment]
Waiting for guidance on the correct branch structure.

**Comment by sandarumk** [comment]
@pandilladivya can you change the base branch to 1.2 rather than develop?

**Pull request merged**